src/fragments/lib/auth/flutter/signin_web_ui/40_private_session.mdx

```diff
@@ -0,0 +1,13 @@
+### Prefer private session during signIn on iOS.
+
+Amplify-flutter uses the amplify-ios library on the iOS platform to facilitate Web UI sign in and other Auth functionality. See the [amplify-ios Web UI documentation](/lib/auth/signin_web_ui/q/platform/ios/#prefer-private-session-during-signin) for details on how amplify-ios manages the interaction between the application and the Web UI.
```

Comment:
I think I prefer "Amplify Flutter" or "amplify-flutter". Not sure if there are guidelines on how to refer to these projects.

Reply:
+1